### PR TITLE
Add Iris LM series of keyboards

### PR DESCRIPTION
**Pull request by nooges**
## Description

Adds Iris LM-G and Iris LM-K PCBs for the Iris LM case.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris_lm/g1/config.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#define SERIAL_USART_RX_PAL_MODE 7
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_PIN_SWAP
+
+#define USB_VBUS_PIN C6
+#define SPLIT_HAND_PIN A0
+
+/* Defines for the RGB matrix */
+#ifdef RGB_MATRIX_ENABLE
+#   define NOP_FUDGE 0.4
```
Comment (waffle87) [inline keyboards/keebio/iris_lm/g1/config.h:20]: This shouldn't be needed.
```suggestion
```

File: keyboards/keebio/iris_lm/g1/keyboard.json [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+    "keyboard_name": "Iris LM-G Rev. 1",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "extrakey": true,
+        "mousekey": true,
+        "rgb_matrix": true
+    },
+    "encoder": {
+        "enabled": true,
```
Comment (waffle87) [inline keyboards/keebio/iris_lm/g1/keyboard.json:11]:
```suggestion
        "rgb_matrix": true,
        "encoder": true
    },
    "encoder": {
```

File: keyboards/keebio/iris_lm/k1/config.h [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+#define SERIAL_USART_RX_PAL_MODE 7
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_PIN_SWAP
+
+#define USB_VBUS_PIN C6
+#define SPLIT_HAND_PIN A0
+
+/* Defines for the RGB matrix */
+#ifdef RGB_MATRIX_ENABLE
+#   define NOP_FUDGE 0.4
```
Comment (waffle87) [inline keyboards/keebio/iris_lm/k1/config.h:20]:
```suggestion
```

File: keyboards/keebio/iris_lm/g1/keyboard.json [hunk 4]
```diff
@@ -0,0 +1,253 @@
+{
+    "keyboard_name": "Iris LM-G Rev. 1",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
```
Comment (waffle87) [inline keyboards/keebio/iris_lm/g1/keyboard.json:5]:
```suggestion
    "features": {
        "bootmagic": true,
```

File: keyboards/keebio/iris_lm/k1/keyboard.json [hunk 5]
```diff
@@ -0,0 +1,238 @@
+{
+    "keyboard_name": "Iris LM-K Rev. 1",
+    "bootloader": "stm32-dfu",
+    "diode_direction": "COL2ROW",
+    "features": {
```
Comment (waffle87) [inline keyboards/keebio/iris_lm/k1/keyboard.json:5]:
```suggestion
    "features": {
        "bootmagic": true,
```

File: keyboards/keebio/iris_lm/g1/rules.mk
Comment (waffle87) [inline keyboards/keebio/iris_lm/g1/rules.mk:1]: This file should be removed:
- `SERIAL_DRIVER` should be migrated to `keyboard.json` under `split.serial.driver`
- These are the default values for EEPROM/Wear-leveling drivers

File: keyboards/keebio/iris_lm/k1/rules.mk
Comment (waffle87) [inline keyboards/keebio/iris_lm/k1/rules.mk:1]: As above.

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris_lm/g1/config.h (hunk 1 above)
Comment (nooges) [reply keyboards/keebio/iris_lm/g1/config.h]: Ah right, we fixed it the last time

**Review by drashna — APPROVED** [review]

**Review by waffle87 — APPROVED** [review]

**Pull request merged**